src/Chainweb/Version.hs

```diff
@@ -572,4 +572,4 @@ txSilenceEndDate TimedCPM{} = Nothing
 txSilenceEndDate FastTimedCPM{} = Nothing
 txSilenceEndDate Development = Nothing
 txSilenceEndDate Testnet03 = Nothing
-txSilenceEndDate Mainnet01 = Just [timeMicrosQQ| 2019-12-05T00:00:00.0 |]
+txSilenceEndDate Mainnet01 = Just [timeMicrosQQ| 2019-12-06T00:00:00.0 |]
```

Comment:
We can't change this. This would be a hard fork.

Reply:
This is a property of the `ChainwebVersion`. Any change of the `chainwebVersion` or its properties is a fork.

Reply:
I will add a second one.

Reply:
The second one, that remarks the end of the soft reject of tx should be larger than `txSilenceEndDate`. In terms of above timeline `txSilenceEndDate == d1` and date for ending the soft reject is `d2`.